firestore-send-email/extension.yaml

```diff
@@ -110,9 +110,8 @@ params:
     validationErrorMessage: Must be a valid email address or valid name plus email address
     required: true
     description: >-
-      The email address to use as the sender's address (if it's not specified in the added email document). 
-      You can optionally include a name with the email address (Friendly Firebaser <foobar@example.com>).
```

Comment:
@agau4779 Is it possible to use back-ticks in the description? E.g:

> You can optionally include a name with the email address (`Friendly Firebaser <foobar@example.com>`).

Reply:
Yeah that's what we were thinking about doing for our next release, could you test that out and make sure it looks good via the CLI and Console?